### PR TITLE
5th line added

### PR DESCRIPTION
**Pull request by malihasultanasathi**
hey bahmedraaz, I added the 5th line. Can you please check it?

### CODE REVIEW

File: 4.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ first line is added
 2nd line is added
 3rd line added
 4th line added
+I added the fifth line
```
Comment (bahmedraaz) [inline 4.txt]: nice job. Can you also add the 6th line. I appreciate your support.

**Review by bahmedraaz — COMMENTED** [review]
gteat!! also if you can please add the 6th line

**Comment by malihasultanasathi** [comment]
6th line added

**Comment by malihasultanasathi** [comment]
6th line added

**Pull request closed without merging**